cabal.project

```diff
@@ -13,6 +13,16 @@ source-repository-package
     location: https://github.com/peti/cabal-plan
     tag: 894b76c0b6bf8f7d2f881431df1f13959a8fce87
 
+source-repository-package
+    type: git
+    location: https://github.com/mpickering/hie-bios
+    tag: a852cffb0deb306cd1e1fe02aa50d50c77aeb15d
```

Comment:
Unneeded, afaict?

Reply:
Needed for https://github.com/mpickering/hie-bios/commit/a852cffb0deb306cd1e1fe02aa50d50c77aeb15d